.circleci/config.yml

```diff
@@ -331,6 +370,35 @@ jobs:
               $host.SetShouldExit(1)
             }
 
+  merge-release-branch:
+    parameters:
+      e:
+        type: executor
+    executor: << parameters.e >>
+    steps:
+      - slack/notify:
+          channel: 'pdt_releases'
+          color: '#FFDEAD'
+          message: 'Merging release branch into main'
+      - ssh-config
+      - checkout
+      - prepare-release-environment-variables
+      - merge-validation
+      - run:
+          name: 'Rebasing main off of release branch'
+          command: |
+            git checkout main
+            echo "Rebasing main off of release branch ${RELEASE_VERSION}"
+            git rebase release/v${RELEASE_VERSION}
```

Comment:
Explanation for using rebase: see [this](https://www.atlassian.com/git/tutorials/merging-vs-rebasing) article

We want to use rebase when pulling in commits from the release branch because we do want to rewrite the git history. Everything that is new from the release branch should be put on "top" of what is currently in main.